stellar/utils.js

```diff
@@ -173,11 +214,37 @@ async function estimateCost(tx, server) {
     };
 }
 
+const getAmplifierVerifiers = async (config, chainAxelarId) => {
+    const client = await CosmWasmClient.connect(config.axelar.rpc);
+    const { id: verifierSetId, verifier_set: verifierSet } = await client.queryContractSmart(
+        config.axelar.contracts.MultisigProver[chainAxelarId].address,
+        'current_verifier_set',
+    );
+    const signers = Object.values(verifierSet.signers);
+
+    const weightedSigners = signers
+        .map((signer) => ({
+            signer: arrayify(`0x${signer.pub_key.ed25519}`),
+            weight: Number(signer.weight),
+        }))
+        .sort((a, b) => hexlify(a.signer).localeCompare(hexlify(b.signer)));
+
+    return {
+        signers: weightedSigners,
+        threshold: Number(verifierSet.threshold),
+        nonce: ethers.utils.hexZeroPad(BigNumber.from(verifierSet.created_at).toHexString(), 32),
+        verifierSetId,
+    };
+};
+
 module.exports = {
```

Comment:
Keep it in order for better code readability:
```suggestion
module.exports = {
    getNetworkPassphrase,
    addBaseOptions,
    buildTransaction,
    prepareTransaction,
    sendTransaction,
    broadcast,
    getAssetCode,
    getWallet,
    estimateCost,
    getAmplifierVerifiers,
};
```